
<script>
    //throw
    try{
        throw "Error occured";
    }
    catch(ex){
        alert(ex);
    }
</script>